src/main/java/com/minecolonies/achievements/ModAchievements.java

```diff
@@ -0,0 +1,118 @@
+package com.minecolonies.achievements;
+
+import net.minecraft.stats.Achievement;
+import net.minecraftforge.common.AchievementPage;
+
+/**
+ * Achievement collection
+ *
+ * @author Isfirs
+ * @since 0.2
+ */
+public final class ModAchievements
+{
+
+    // Achievements
+    /**
+     * Place a supply chest
+     */
+    public static final Achievement achievementGetSupply      = new AchievementGetSupply("supply", "supply", -2, -2).registerStat();
+    /**
+     * Use the building tool
+     */
+    public static final Achievement achievementWandOfbuilding = new AchievementWandOfBuilding("wandofbuilding", "wandofbuilding", 0, -2).registerStat();
+
+    // Buildings
+    /**
+     * Place a townhall
+     */
+    public static final Achievement achievementBuildingTownhall = new AchievementBuildingTownhall("townhall", "townhall", -2, 0).registerStat();
+
+    /**
+     * Upgrade a builder to lv 1
+     */
+    public static final Achievement achievementBuildingBuilder   = new AchievementBuildingBuilder("upgrade.builder.first", "upgrade.builder.first", 0, 1).registerStat();
+    /**
+     * Max out a builder
+     */
+    public static final Achievement achievementUpgradeBuilderMax = new AchievementUpgradeBuilderMax("upgrade.builder.max", "upgrade.builder.max", 2, 1).registerStat();
+
+    /**
+     * Upgrade a builder to lv 1
+     */
+    public static final Achievement achievementBuildingColonist   = new AchievementBuildingColonist("upgrade.colonist.first", "upgrade.colonist.first", 0, 2).registerStat();
+    /**
+     * Max out a builder
+     */
+    public static final Achievement achievementUpgradeColonistMax = new AchievementUpgradeColonistMax("upgrade.colonist.max", "upgrade.colonist.max", 2, 2).registerStat();
+
+    /**
+     * Upgrade a lumberjack to lv 1
+     */
+    public static final Achievement achievementBuildingLumberjack   = new AchievementBuildingLumberjack("upgrade.lumberjack.first", "upgrade.lumberjack.first", 0, 3).registerStat();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 181 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00103)